examples/integration_test.py

```diff
@@ -45,14 +45,16 @@
 except ImportError as e:
     with_progress = False
 
+testconf = 'tests/testconf-example.json'
```

Comment:
testconf.json, the example file is an example / template, it shouldnt be used directly without editing.

Let it default to tests/testconf.json and if it does not exist tell the user to write their own based on testconf-example.json